package.json

```diff
@@ -66,18 +68,18 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "enzyme": "^3.2.0",
-    "enzyme-adapter-react-16": "^1.1.0",
-    "enzyme-to-json": "^3.2.2",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "enzyme-to-json": "^3.4.3",
     "eslint": "^4.13.1",
     "http-server": "^0.10.0",
     "husky": "^2.7.0",
     "jest": "^21.2.1",
     "lint-staged": "^8.2.1",
     "prettier": "^1.9.2",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-test-renderer": "^16.2.0",
+    "react": "^16.11.0",
```

Comment:
Do we really need to se thist to 16.11.0 ? Anyone using react below this version will have a warning to install this version also. I think we cant set it to a version from which the new context API is available ?

Reply:
This is just the dev dependency. The peer depency remains as `"react": "0.14.x || ^15.0.0 || ^16.0.0",` since we're using `create-react-context` to ponyfill the new context API for versions of React prior to v16.3.0.